config/locales/telephony/en.yml

```diff
@@ -15,7 +15,7 @@ en:
         code expires in %{expiration} minutes.
     confirmation_ipp_enrollment_result:
       sms: |-
-        %{app_name}: You attempted to verify your identity at a Post Office on %{proof_date}. Check your email for your result.
+        %{app_name}: You visited the Post Office on %{proof_date}. Check your email for your result. Not you? Report this right away: %{contact_number}
```

Comment:
Update message does not contain YOUR.

doc reads: Check email for your result.
you here: Check **your** email for your result.